Bottom: c08fb825a11a15fa259a3ae6de799b869a4a765c
Top:    197fa9d6089cf3f1050a93a09906554fdca6fb60
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-29 17:48:49 +0100

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 994aeff..5502c29 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1680,7 +1680,7 @@ Gather (VNODE * start, PLINE ** result, J_Rule v_rule, DIRECTION initdir)
       if ((newn = poly_CreateNodeFull (cur->point, (dir == FORW) ? cur->is_round : cur->next->is_round,
                                                    (dir == FORW) ? cur->cx       : cur->next->cx,
                                                    (dir == FORW) ? cur->cy       : cur->next->cy,
-                                                   (dir == FORW) ? cur->radius   : cur->next->radius)) == NULL) /* XXX: DIRECTION - might we need to query the nextious point for arc details ?? */
+                                                   (dir == FORW) ? cur->radius   : cur->next->radius)) == NULL) /* XXX: DIRECTION - might we need to query the previous point for arc details ?? */
         return err_no_memory;
       if (!*result)
 	{
